onnxruntime/python/tools/quantization/E2E_example_model/CalTableFlatBuffers/KeyValue.py

```diff
@@ -0,0 +1,40 @@
+# automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
why is this file under E2E_example_model ?

Reply:
move CalTableFlatBuffers to quantization tool root folder.